unittests/data/type_traits.hpp

```diff
@@ -544,13 +586,14 @@ namespace yes{
     struct x{
         x(){}
     };
+#ifdef __castxml__
+    typedef details::const_item const_item;
+    typedef details::const_container const_container;
+#endif
```

Comment:
I just opened a PR to bump the xml version number: https://github.com/CastXML/CastXML/pull/57
Just wrap these tests with this revision number. No need to write multiple tests for older versions, this will get too messy.

When running the tests locally, I'm always running against the newest castxml. I'll update the binaries used for the Travis build later so that these tests are also run. My hope is that people will try to move to the latest CastXML versions as fast as possible.